pramen-py/src/pramen_py/runner/runner_transformation.py

```diff
@@ -67,16 +69,25 @@ def overwrite_info_dates(
     return config
 
 
-def discover_transformations(
-    path: pathlib.Path = TRANSFORMATIONS_DIR,
-) -> Generator[Type[T_TRANSFORMATION], None, None,]:
+def get_transformations_dir(dir_key: str, envs: Env) -> pathlib.Path:
```

Comment:
```suggestion
def get_transformations_dir(dir_key: str, env: Env) -> pathlib.Path:
```
this is more typical naming for env wrapper